src/amoc_api_test_handler.erl

```diff
@@ -11,34 +11,34 @@
 
 -type state() :: list().
 
--spec init(tuple(), cowboy:req(), state()) -> {upgrade, protocol, cowboy_rest}.
+-spec init(tuple(), cowboy_req:req(), state()) -> {upgrade, protocol, cowboy_rest}.
 init({tcp, http}, _Req, _Opts) ->
     {upgrade, protocol, cowboy_rest}.
 
--spec rest_init(cowboy:req(), [atom()]) -> {ok, cowboy:req(), state()}.
+-spec rest_init(cowboy_req:req(), [atom()]) -> {ok, cowboy_req:req(), state()}.
 rest_init(Req, _) ->
     {ok, Req, []}.
 
--spec allowed_methods(cowboy:req(), state()) -> 
-        {[binary()], cowboy:req(), state()}.
+-spec allowed_methods(cowboy_req:req(), state()) -> 
+        {[binary()], cowboy_req:req(), state()}.
 allowed_methods(Req, State) ->
     {[<<"GET">>, <<"POST">>], Req, State}.
 
--spec content_types_accepted(cowboy:req(), state()) -> 
-        {[tuple()], cowboy:req(), state()}.
+-spec content_types_accepted(cowboy_req:req(), state()) -> 
+        {[tuple()], cowboy_req:req(), state()}.
 content_types_accepted(Req, State) ->
     {[
         {<<"application/json">>, from_json}
     ], Req, State}.
 
--spec content_types_provided(cowboy:req(), state()) -> 
-        {[tuple()], cowboy:req(), state()}.
+-spec content_types_provided(cowboy_req:req(), state()) -> 
+        {[tuple()], cowboy_req:req(), state()}.
 content_types_provided(Req, State) ->
     {[
         {<<"application/json">>, to_json}
     ], Req, State}.
 
--spec from_json(cowboy:req(), state()) -> {halt | false, cowboy:req(), state()}.
+-spec from_json(cowboy_req:req(), state()) -> {halt | false, cowboy_req:req(), state()}.
```

Comment:
Line 41 is too long: -spec from_json(cowboy_req:req(), state()) -> {halt | false, cowboy_req:req(), state()}..